core/storage/topic/gae_models.py

```diff
@@ -55,8 +55,13 @@ class TopicModel(base_models.VersionedModel):
     # This consists of the list of additional (non-canonical) story ids that
     # are part of this topic.
     additional_story_ids = ndb.StringProperty(repeated=True, indexed=True)
-    # This consists of the full list of skill ids that are part of this topic.
-    skill_ids = ndb.StringProperty(repeated=True, indexed=True)
+    # This consists of the list of uncategorized skill ids that are not part of
+    # any subtopic.
+    uncategorized_skill_ids = ndb.StringProperty(repeated=True, indexed=True)
+    # The list of subtopics that are part of the topic.
+    subtopics = ndb.JsonProperty(repeated=True, indexed=False)
+    # The schema version of the subtopic dict.
+    schema_version = ndb.IntegerProperty(required=True, indexed=True)
```

Comment:
subtopic_schema_version?

(not sure if should be subtopic_* or subtopics_*; follow existing conventions)

Reply:
Done.